docs/source/gateway/index.rst

```diff
@@ -1,6 +1,1049 @@
+
+As of now, the MLflow AI Gateway supports the following providers:
+
+* **openai**: This is used for models offered by `OpenAI <https://platform.openai.com/>`_.
+* **anthropic**: This is used for models offered by `Anthropic <https://docs.anthropic.com/claude/docs>`_.
+* **cohere**: This is used for models offered by `Cohere <https://docs.cohere.com/docs>`_.
+* **azure** / **azuread**: This is used for model services offered by `Azure <https://learn.microsoft.com/en-gb/azure/cognitive-services/openai/>`_.
+
+More providers are being added continually. Check the latest version of the MLflow AI Gateway Docs for the
+most up-to-date list of supported providers.
+
+Remember, the provider you specify must be one that the MLflow AI Gateway supports. If the provider
+is not supported, the Gateway will return an error when trying to route requests to that provider.
+
+Authentication
+--------------
+
+In many organizations, managing API keys can be a cumbersome and error-prone task. With a high number
+of users, sharing keys becomes a security risk and individual key management can become overwhelming.
+The MLflow AI Gateway helps mitigate these issues by centrally managing the keys and allowing users to
+access the service without ever needing to handle API keys directly.
+
+Here's a brief overview of how the authentication process works:
+
+* **Key Management**: As an administrator, you'll set up the MLflow AI Gateway and input your API keys (as environment variables or directly into the configuration files). The Gateway securely stores these keys and uses them to authenticate requests with the service provider.
+
+* **User Access**: Instead of providing individual API keys to every user in your organization, you'll give them access to the MLflow AI Gateway. Users then send their requests directly to the Gateway, which acts as a proxy to the service provider.
+
+* **Request Handling**: When a user sends a request to the MLflow AI Gateway, it takes the request, adds the necessary authentication (using the API keys it manages), and forwards the request to the correct provider (as specified in the configuration files).
+
+* **Response Forwarding**: The Gateway receives the response from the provider and then sends this response back to the user.
+
+By centralizing key management, the MLflow AI Gateway dramatically reduces the risk of keys being
+lost, misused, or accessed by unauthorized individuals. This approach also simplifies the process
+for end users - they no longer need to worry about managing API keys and can instead focus on
+making requests and working with the responses.
+
+This centralized system allows you to change providers or models easily. If you need
+to switch models or update API keys, you can do so in a central location without requiring any
+changes from your end users. This makes the MLflow AI Gateway a robust and versatile solution for
+API key management and service integration within your organization.
```

Comment:
I think we can remove this section for now. The routes / providers / models / configuration section should be enough